tests/fe/fe_values_view_30.cc

```diff
@@ -109,7 +109,7 @@ void test (const Triangulation<dim> &tr,
 
   MappingQGeneric<dim> mapping(1);
 
-  const QGauss<dim> quadrature(2);
+  const QGauss<dim> quadrature(fe.degree+2);
```

Comment:
Hm, I may have missed this -- `fe_values_view_30` calls the `project()` function, which now produces different output than it did before? Or do I misunderstand your comment?